tests/behat/features/capabilities/group/group-create-secret.feature

```diff
@@ -0,0 +1,182 @@
+    And I should see "Disclosed"
+    And I should see "1 member"
+    And I should see "Joined"
+    And I should not see the link "Read more"
+
+    And I should see "Test secret group" in the "Hero block"
+    And I should see the button "Joined"
+    And I should see the link "Edit group" in the "Hero block"
+    And I should see "Disclosed" in the "Hero block"
+    And I should see "Padangstraat 11" in the "Hero block"
+    And I should see "7556SP" in the "Hero block"
+
+  # Create a post inside the secret group, visible to group members only
+    When I click "Stream"
+    And I fill in "Say something to the group" with "This is a secret group post."
+    And I select post visibility "Group members"
+    And I press "Post"
+    Then I should see the success message "Your post has been posted."
+    And I should see "This is a secret group post."
+
+  # Create a topic inside the secret group
+    When I click "Topics"
+    And I should see the link "Create Topic" in the "Sidebar second"
+    And I click "Create Topic"
+    When I fill in the following:
+      | Title | Test secret group topic |
+    And I fill in the "edit-body-0-value" WYSIWYG editor with "Body description text"
+    And I click radio button "Discussion"
+    And I press "Save"
+    And I should see "Test secret group topic"
+
+  # Create an event inside the secret group
+    And I click "Test secret group"
+    When I click "Events"
+    And I should see the link "Create Event" in the "Sidebar second"
+    And I click "Create Event"
+    And I fill in the following:
+      | Title | Test secret group event |
+      | edit-field-event-date-0-value-date | 2025-01-01 |
+      | edit-field-event-date-end-0-value-date | 2025-01-01 |
+      | Time  | 11:00:00    |
+      | Location name       | Technopark |
+    And I fill in the "edit-body-0-value" WYSIWYG editor with "Body description text."
+    And I press "Save"
+    And I should see "Test secret group event"
+
+  # Lets add another user on the Manage members tab.
+    When I click "Test secret group"
+    And I click "Manage members"
+    And I click "Add members"
+    And I fill in "SecretGroup User Two" for "Select members to add"
+    And I press "Save"
+    Then I click "Members"
+    And I should see "SecretGroup User Two"
+    And I logout
+
+  # Now login as user two.
+    Given I am logged in as "SecretGroup User Two"
+    And I am on "/all-groups"
+    Then I should see "Test secret group"
+    When I click "Test secret group"
+    And I should see the button "Joined"
+    And I should see "Test secret group topic"
+    And I should see "Test secret group event"
+    And I should see "This is a secret group post."
+    And I open and check the access of content in group "Test secret group" and I expect access "allowed"
+    And I logout
+    And I open and check the access of content in group "Test secret group" and I expect access "denied"
+
+  # As a non-member of the secret group, I should not see anything.
```

Comment:
I think we should refactor this into the individual scenario's that we created above and just say `Given that I am logged in as "authenticated user"` in those tests, then assert what we want.